gdlsc.go

```diff
@@ -1,104 +1,194 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
-	"os/exec"
+	"path"
+	"regexp"
 	"strings"
+
+	"github.com/ryanuber/go-license"
 )
 
-// URL represents path a github repos license
-var URL = "https://api.github.com/repos/%s/license?access_token=" + os.Getenv("ACCESS_TOKEN")
+const vendorDir = "vendor"
 
-//types for message to be printed
-type message struct {
-	Dependency string
-	License    string
-	Link       string
+// Node describes a directory in the vendor tree
+type Node struct {
```

Comment:
Just FYI this does not need to be public, so the name would better be lowercase.